### DISCUSSION TITLE
Can I bind specific presets to certain devices?

### DISCUSSION THREAD

**Discussion by nadiamoe** [wwmm/easyeffects#1082] [Q&A] [answered]
I change my output devices quite often (from speakers to BT headphones) and they have dramatically different freq response curves. I have some presets including equalization for each one, and I'd love if EasyEffects could switch from one to another based on which device is the current default.

I _vaguely_ remember this being a feature when pulseaudio was being used, but I'm not positive about it. Is this still an option I cannot find? If not, is it in the roadmap somehow?

Thanks!

**Comment by wwmm** [comment, marked as answer]
You can do that
![Screenshot from 2021-08-10 10-58-06](https://user-images.githubusercontent.com/16225172/128880239-0f42d29b-7589-48b7-9692-f68348be3be3.png)

It works with most hardware. But there are some corner cases that still need solutions https://github.com/wwmm/easyeffects/issues/1051

**Reply by nadiamoe** [reply]
That's awesome, I totally missed it. Thanks! ❤️

**Comment by Purgenta889** [comment]
BTW, for anyone encountering this thread in the future (as of July 28th, 2026), this has been moved to the "Presets" page:
<img width="893" height="234" alt="image" src="https://github.com/user-attachments/assets/5d453f5d-4693-494a-82d1-0d3abd63e977" />
